README.md

```diff
@@ -1 +1,66 @@
-# loaderboard-proj
\ No newline at end of file
+# loaderboard-proj
+In this project I built a Leaderboard list app, where the following was implemented in the project
+- Created a new game using the API.
+- saved the ID of my game that will be returned by API.
+- Implemented the "Refresh" button that gets all scores for the game created from the API.
+- Implemented the form "Submit" button that will save scores for the game.
+- Used arrow functions instead of the function keyword.
+- Used async and await JavaScript features to consume the API.
+## Built With
+
+- HTML
+- CSS
+-javascript
+- Linters
+-webpack
+
+### Prerequisites
+
+- A browser
+- A code editor example VS code
+
+### Setup
+
+- A computer with a browser
+- copy  the github link for the repository Leaderboard list app, using webpack and ES6 modules. 
+### Install
+
+- Copy the github link for the repository
+- Open the git terminal and type git clone then paste the repository link
+
+### Run tests
+
+- It has been tested and it is fully responsive
```

Comment:
- [OPTIONAL] I think it's better to use this item to explain how a user that clones your repo runs the tests created by you during the test activities using tools like `jest` or `rspec`. If you're not doing tests, removing the test item is better.